pkg/vsphere/session/session.go

```diff
@@ -327,6 +330,14 @@ func (s *Session) Populate(ctx context.Context) (*Session, error) {
 		errs = append(errs, fmt.Sprintf("Failure finding cluster (%s): %s", s.ClusterPath, err.Error()))
 	} else {
 		op.Debugf("Cached cluster: %s", s.ClusterPath)
+		// if we have a cluster lets get DRS Status
+		if s.Cluster != nil && s.Cluster.Reference().Type == "ClusterComputeResource" {
```

Comment:
constant for the magic value here. Looks like it lines up with the one above. I think we have a constants file somewhere :/

Reply:
Maybe just add that constant to the new Cluster package that you created. :)

Reply:
I'm not using the cluster addition because of go import madness...I think we go constant crazy...anyhoo...will rectify when I clean up the DRS check (which requires Cluster)